lib/internal/test_runner/reporter/tap.js

```diff
@@ -234,9 +236,27 @@ function jsToYaml(indent, name, value) {
       }
     }
 
-    if (typeof errStack === 'string') {
-      const frames = [];
+    const frames = [];
 
+    if (cause?.cause) {
+      const errCause = util.inspect(cause).split('\n').slice(1)
+                      .filter((line) => !StringPrototypeStartsWith(line.trim(), '...') && line)
```

Comment:
It seems like the `&& line` should be `line &&`. Also, I would check for `typeof line === 'string'`